src/application/dataFetcher/AxiosUrlDataFetcherAdapter.ts

```diff
@@ -1,11 +1,16 @@
+import Request from '@core/domain/request/Request';
 import axios from 'axios';
-import UrlDataFetcher from '../../domain/common/port/UrlDataFetcherPort';
-import { HttpError } from '../../domain/common/utils/error';
 
-class AxiosUrlDataFetcherAdapter implements UrlDataFetcher {
-  async fetch(url): Promise<string> {
+import DataFetcher from '@core/domain/common/port/DataFetcherPort';
+import { HttpError } from '@core/domain/common/utils/error';
+import RequestUrlParser from '@core/domain/request/RequestUrlParser';
```

Comment:
Looks like RequestUrlParser is now used in two different places (here and in Response domain object). Should AxiosUrlDataFetcherAdapter internalize this logic instead of calling it?

Reply:
Yes. In current state, RequestUrlParser implements some features that are distinct to the RequestExecutor. It will be refactored later, as now we need this in master to deliver random data provider.